cmd/dockerRegistry.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var DOCKER_REGISTRY_NAME string = "docker-image-reg"
+
+func installDockerRegistry() {
+	log.Info("Starting " + DOCKER_REGISTRY_NAME)
+	cmd := exec.Command(
+		"docker",
+		"run",
+		"-d",
+		"-p5000:5000",
+		"--pull=missing",
+		"--name="+DOCKER_REGISTRY_NAME,
+		"--restart=always",
+		"registry:2",
+	)
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Infof("%s", out)
+		log.Fatalf("failed to start "+DOCKER_REGISTRY_NAME+": %v", err)
+	}
+
+	// connect to kind network
+	cmd = exec.Command(
+		"docker",
+		"network",
+		"connect",
+		"kind",
+		DOCKER_REGISTRY_NAME,
+	)
+
+	out, err = cmd.CombinedOutput()
+	if err != nil {
+		log.Infof("%s", out)
+		log.Fatalf("failed to add "+DOCKER_REGISTRY_NAME+"to the kind network: %v", err)
```

Comment:
if we fail here and try and rerun the calling command I assume it wound fail since the docker container is already running

Reply:
added a check to see if the docker container is running before trying to construct another